src/zenml/integrations/openai/hooks/open_ai_failure_hook.py

```diff
@@ -111,4 +133,4 @@ def openai_gpt4_alerter_failure_hook(
     Args:
         exception: The exception that was raised.
     """
-    openai_alerter_failure_hook_helper(exception, "gpt-4")
+    openai_alerter_failure_hook_helper(exception, "gpt-4")
```

Comment:
```suggestion
    openai_alerter_failure_hook_helper(exception, "gpt-4o")
```

I guess this is the usual default for people these days...